Tools/WebKitTestRunner/cocoa/TestWebsiteDataStoreDelegate.mm

```diff
@@ -155,8 +155,10 @@ - (NSData *)webCryptoMasterKey
 {
     // Not so random key
     constexpr size_t keyLength = 16;
-    uint8_t keyBytes[keyLength] = { 1 };
+    // same value as https://github.com/WebKit/WebKit/blob/9854e2932ee221bed4a8260dbc9ec7b0d87189ed/Tools/WebKitTestRunner/TestController.cpp#L133
```

Comment:
This comment is quite likely to get out of date.

// same value as copyWebCryptoMasterKey